contracts/beneficiary.sol

```diff
@@ -12,31 +12,36 @@ import "@zondax/filecoin-solidity/contracts/v0.8/utils/FilAddresses.sol";
 
 import "@zondax/filecoin-solidity/contracts/v0.8/types/CommonTypes.sol";
 
-// import "fevmate/contracts/utils/FilAddress.sol";
+import "fevmate/contracts/utils/FilAddress.sol";
 
 import "./utils/Common.sol";
 import "./utils/Validator.sol";
-import "./utils/FilAddress.sol";
 
 
 /// @author SPex Team
 contract SPexBeneficiary {
-
-    event EventPledgeBeneficiaryToSpex(CommonTypes.FilActorId, address, uint, uint, address, uint8, uint);
-    event EventReleaseBeneficiary(CommonTypes.FilActorId, CommonTypes.FilAddress);
-    event EventLendToMiner(address, CommonTypes.FilActorId, uint);
-    event EventChangeMinerDelegator(CommonTypes.FilActorId, address);
-    event EventChangeMinerMaxDebtAmount(CommonTypes.FilActorId, uint);
-    event EventChangeMinerDisabled(CommonTypes.FilActorId, bool);
-    event EventChangeMinerLoanInterestRate(CommonTypes.FilActorId, uint);
-    event EventChangeMinerReceiveAddress(CommonTypes.FilActorId, address);
-    event EventChangeMinerMaxLenderCount(CommonTypes.FilActorId, uint);
-    event EventChangeMinerMinLendAmount(CommonTypes.FilActorId, uint);
-    event EventRepayment(address, address, CommonTypes.FilActorId, uint, uint);
-    event EventWithdrawRepayment(address, address, CommonTypes.FilActorId, uint, uint);
-    event EventSellLoan(address, CommonTypes.FilActorId, uint, uint);
-    event EventBuyLoan(address, address, CommonTypes.FilActorId, uint, uint, uint);
-    event EventCancelSellLoan(address, CommonTypes.FilActorId);
+    event EventPledgeBeneficiaryToSpex(
+        CommonTypes.FilActorId minerId,
```

Comment:
Seems that params like minerId and other address typed params could be indexed as well (an unwrap might be helpful in case of CommonTypes.FilActorId)